src/Psl/Str/Byte/lowercase.php

```diff
@@ -4,12 +4,14 @@
 
 namespace Psl\Str\Byte;
 
+use function strtolower;
+
 /**
  * Returns the string with all alphabetic characters converted to lowercase.
  *
  * @psalm-pure
  */
-function lowercase(string $lowercase): string
+function lowercase(string $string): string
```

Comment:
Just checked, the reserved keyword `string` is ok as a named param
https://3v4l.org/u0D8K

However, since it is a type - Is this a good name for a variable?

Reply:
this is what php-src is using now. i can't think of a a better alternative.